test/common/TestBed.cpp

```diff
@@ -220,21 +220,37 @@ namespace RcclUnitTesting
     }
   }
 
-  void TestBed::ExecuteCollectives()
+  void TestBed::ExecuteCollectives(std::vector<int> const &currentRanks)
   {
     int const cmd = TestBedChild::CHILD_EXECUTE_COLL;
     ++TestBed::NumTestsRun();
 
-    // Send ExecuteColl command to each active child process
+    std::vector<std::vector<int>> childToRankMap;
     for (int childId = 0; childId < this->numActiveChildren; ++childId)
     {
-      PIPE_WRITE(childId, cmd);
+      childToRankMap.push_back({});
+    }
```

Comment:
Would be easier to just resize at construction.
std::vector<std::vector<int>> ranksPerChild(this->numActiveChildren);